### ISSUE TITLE
Drush 8.4.10 "make" command throws fatal error if makefile is YML and drush is phar

### ISSUE THREAD

**Issue by ssudhak** [drush-ops/drush#5140]
PHP Fatal error:  Uncaught Error: Class "Drush\Internal\Symfony\Yaml\Yaml" not found in phar:///usr/local/bin/drush8/lib/Drush/Make/Parser/ParserYaml.php:26
Stack trace:
#0 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(662): Drush\Make\Parser\ParserYaml::parse('core: 7.x\napi: ...')
#1 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(40): _make_determine_format('core: 7.x\napi: ...')
#2 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(15): _make_parse_info_file('make.yml')
#3 phar:///usr/local/bin/drush8/commands/make/make.drush.inc(309): make_parse_info_file('make.yml')
#4 phar:///usr/local/bin/drush8/includes/command.inc(422): drush_make('make.yml')
#5 phar:///usr/local/bin/drush8/includes/command.inc(231): _drush_invoke_hooks(Array, Array)
#6 phar:///usr/local/bin/drush8/includes/command.inc(199): drush_command('make.yml')
#7 phar:///usr/local/bin/drush8/lib/Drush/Boot/BaseBoot.php(67): drush_dispatch(Array)
#8 phar:///usr/local/bin/drush8/includes/preflight.inc(67): Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#9 phar:///usr/local/bin/drush8/includes/startup.inc(465): drush_main()
#10 phar:///usr/local/bin/drush8/includes/startup.inc(369): drush_run_main(false, '/', 'Phar detected. ...')
#11 phar:///usr/local/bin/drush8/drush(114): drush_startup(Array)
#12 /usr/local/bin/drush8(14): require('phar:///usr/loc...')
#13 {main}
  thrown in phar:///usr/local/bin/drush8/lib/Drush/Make/Parser/ParserYaml.php on line 26

Fatal error: Uncaught Error: Class "Drush\Internal\Symfony\Yaml\Yaml" not found in phar:///usr/local/bin/drush8/lib/Drush/Make/Parser/ParserYaml.php:26
Stack trace:
#0 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(662): Drush\Make\Parser\ParserYaml::parse('core: 7.x\napi: ...')
#1 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(40): _make_determine_format('core: 7.x\napi: ...')
#2 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(15): _make_parse_info_file('make.yml')
#3 phar:///usr/local/bin/drush8/commands/make/make.drush.inc(309): make_parse_info_file('make.yml')
#4 phar:///usr/local/bin/drush8/includes/command.inc(422): drush_make('make.yml')
#5 phar:///usr/local/bin/drush8/includes/command.inc(231): _drush_invoke_hooks(Array, Array)
#6 phar:///usr/local/bin/drush8/includes/command.inc(199): drush_command('make.yml')
#7 phar:///usr/local/bin/drush8/lib/Drush/Boot/BaseBoot.php(67): drush_dispatch(Array)
#8 phar:///usr/local/bin/drush8/includes/preflight.inc(67): Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#9 phar:///usr/local/bin/drush8/includes/startup.inc(465): drush_main()
#10 phar:///usr/local/bin/drush8/includes/startup.inc(369): drush_run_main(false, '/', 'Phar detected. ...')
#11 phar:///usr/local/bin/drush8/drush(114): drush_startup(Array)
#12 /usr/local/bin/drush8(14): require('phar:///usr/loc...')
#13 {main}
  thrown in phar:///usr/local/bin/drush8/lib/Drush/Make/Parser/ParserYaml.php on line 26
Drush command terminated abnormally due to an unrecoverable error.                                [error]
Error: Uncaught Error: Class "Drush\Internal\Symfony\Yaml\Yaml" not found in
phar:///usr/local/bin/drush8/lib/Drush/Make/Parser/ParserYaml.php:26
Stack trace:
#0 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(662):
Drush\Make\Parser\ParserYaml::parse('core: 7.x\napi: ...')
#1 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(40):
_make_determine_format('core: 7.x\napi: ...')
#2 phar:///usr/local/bin/drush8/commands/make/make.utilities.inc(15):
_make_parse_info_file('make.yml')
#3 phar:///usr/local/bin/drush8/commands/make/make.drush.inc(309):
make_parse_info_file('make.yml')
#4 phar:///usr/local/bin/drush8/includes/command.inc(422): drush_make('make.yml')
#5 phar:///usr/local/bin/drush8/includes/command.inc(231): _drush_invoke_hooks(Array, Array)
#6 phar:///usr/local/bin/drush8/includes/command.inc(199): drush_command('make.yml')
#7 phar:///usr/local/bin/drush8/lib/Drush/Boot/BaseBoot.php(67): drush_dispatch(Array)
#8 phar:///usr/local/bin/drush8/includes/preflight.inc(67):
Drush\Boot\BaseBoot->bootstrap_and_dispatch()
#9 phar:///usr/local/bin/drush8/includes/startup.inc(465): drush_main()
#10 phar:///usr/local/bin/drush8/includes/startup.inc(369): drush_run_main(false, '/', 'Phar
detected. ...')
#11 phar:///usr/local/bin/drush8/drush(114): drush_startup(Array)
#12 /usr/local/bin/drush8(14): require('phar:///usr/loc...')
#13 {main}
  thrown in phar:///usr/local/bin/drush8/lib/Drush/Make/Parser/ParserYaml.php, line 26

**Comment by greg-1-anderson** [comment]
Looks like the Drush\Internal classes are not being added to the .phar

**Comment by greg-1-anderson** [comment]
Hm, no, it's not that:
```
./drush.phar ev 'var_export(class_exists("\Drush\Internal\Symfony\Yaml\Yaml"));'
```
Not sure what the problem is. PRs welcome, if anyone isolates the problem.

**Comment by yorkhu** [comment]
We have found that [commit](https://github.com/drush-ops/drush/commit/269a599744e9237a822ed3dd0363f2aa4b8e0826) fixed the issue for us.
Could we have a release that includes that last commit?

**Comment by ssudhak** [comment]
@greg-1-anderson Is it possible to make a release with the above mentioned commit

**Comment by nikolay-jobiqo** [comment]
I have issue executing `drush vget` on drush 8.4.10. And this version is used on latest ddev 1.19.2. Is it possible to make release?

**Comment by greg-1-anderson** [comment]
I just released [Drush 8.4.11](https://github.com/drush-ops/drush/releases/tag/8.4.11).

@nikolay-jobiqo This release only has one change over 8.4.10; I am not sure this will have any effect on your `drush vget` problem. If it does not, please open a new issue with more detailed information.

**Comment by nikolay-jobiqo** [comment]
@greg-1-anderson thanks a lot for quick relase. 
`drush vget` is working for me on drush 8.4.11.
